lib/spack/spack/environment.py

```diff
@@ -142,18 +141,20 @@ def activate(
             cmds += 'set prompt="%s ${prompt}";\n' % prompt
     else:
         if os.getenv('TERM') and 'color' in os.getenv('TERM') and prompt:
-            prompt = colorize('@G{%s} ' % prompt, color=True)
+            prompt = r"\[\033[0;92m\]{0}\[\033[0m\]".format(prompt)
```

Comment:
Can we keep the fix inside the `colorize` function, maybe adding a new parameter to it if needed?

Reply:
I'll take a look!

Reply:
So I'm looking at llnl/util/tty/color.py -- specifically the `colorize()` function on line 189 and  the regular expression `color_re` defined on line 94. My regex knowledge is minimal and it looks like hieroglyphics to me. Out of an abundance of caution I am reluctant to be the one to drop a fix inside `colorize()` . Perhaps someone else would feel more comfortable making the change and having it be contingent on the presence of a keyword argument ?